### DISCUSSION TITLE
Build fails due to "Missing package product 'FirebaseStorageSwift-Beta'" using SPM

### DISCUSSION THREAD

**Discussion by radcli14** [firebase/firebase-ios-sdk#12679] [Q&A] [answered]
Per title, when I attempt to build my project I see 17 error messages similar to the above, all referencing Beta versions of some package product.

This is also accompanied by a warning "skipping cache due to an error: https://github.com/firebase/firebase-ios-sdk.git: The repository could not be found. Make sure a valid repository exists at the specified location and try again."

Does anyone know what is causing this, and how it may be corrected? I have attached logs. A few additional details:

- M2 Macbook Air, Sonoma 14.4.1, XCode 15.3
- Using Swift Package Manager, Firebase version set to 10.23.1 up to next major version
- Project is utilizing Compose Multiplatform

[Add Package_2024-04-02T16-31-39.txt](https://github.com/firebase/firebase-ios-sdk/files/14838720/Add.Package_2024-04-02T16-31-39.txt)

[Build iosApp_2024-04-02T16-37-06.txt](https://github.com/firebase/firebase-ios-sdk/files/14838686/Build.iosApp_2024-04-02T16-37-06.txt)

**Comment by ncooke3** [comment, marked as answer]
In Firebase 10, the project should only have a dependency on `FirebaseStorage`. The dependency on `FirebaseStorageSwift-Beta` can be removed. Imports of `FirebaseStorageSwift` should be changed to `FirebaseStorage`.

**Reply by radcli14** [reply]
Thanks for responding. Do you how I can trace down where that dependency is being generated, and remove it? I certainly didn't create it myself, at least not knowingly.

**Reply by ncooke3** [reply]
If you go to the target's _General_ settings, and scroll down to _Frameworks, Libraries, and Embedded Content_ section. I would think that there should be a row that says `FirebaseStorageSwift-Beta`. This row can then be deleted.

**Reply by radcli14** [reply]
Great, I found it, and was able to build successfully after deleting all of the "Beta" dependencies. Thank you for the help!
